cmd/registry/cmd/export/yaml_test.go

```diff
@@ -57,6 +58,8 @@ func TestExportYAML(t *testing.T) {
 		cmd := Command()
 		args := []string{"yaml", r}
 		cmd.SetArgs(args)
+		out := bytes.NewBuffer(make([]byte, 0))
```

Comment:
Can this be replaced with io.Discard?

Reply:
Good point, but I think that's going in the wrong direction -- we should probably be verifying the output in some way.